packages/connectivity/connectivity/example/lib/main.dart

```diff
@@ -144,13 +147,19 @@ class _MyHomePageState extends State<MyHomePage> {
         } on PlatformException catch (e) {
           print(e.toString());
           wifiBSSID = "Failed to get Wifi BSSID";
+        } on MissingPluginException catch (e) {
+          print(e.toString());
+          wifiBSSID = "Location permissions not enabled";
         }
 
         try {
           wifiIP = await _connectivity.getWifiIP();
         } on PlatformException catch (e) {
           print(e.toString());
           wifiIP = "Failed to get Wifi IP";
+        } on MissingPluginException catch (e) {
```

Comment:
Is this catch needed? I thought that the exception could only happen for the `getWifiName` and `getWifiBSSID` methods.... while `getWifiIP` would continue working as normal?

Reply:
you're right! took that out